src/nginxconfig/templates/global_sections/security.vue

```diff
@@ -102,6 +102,41 @@ THE SOFTWARE.
                 </div>
             </div>
         </div>
+
+        <div class="field is-horizontal">
+            <div class="field-label">
+                <label class="label">security.txt</label>
+            </div>
+            <div class="field-body">
+                <div class="field">
+                    <div :class="`control${securityTxt ? ' is-changed' : ''}`">
+                        <div class="checkbox">
+                            <PrettyCheck v-model="securityTxt" class="p-default p-curve p-fill p-icon">
+                                <i slot="extra" class="icon fas fa-check"></i>
+                                {{ i18n.common.enable }}
+                            </PrettyCheck>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div v-if="$props.data.securityTxt.computed" class="field is-horizontal">
+            <div class="field-label">
+                <label class="label">security.txt path</label>
+            </div>
+            <div class="field-body">
+                <div class="field">
+                    <div :class="`control${securityTxtChanged ? ' is-changed' : ''}`">
+                        <input v-model="securityTxtPath"
+                               class="input"
+                               type="text"
+                               :placeholder="`en, es, ru`"
```

Comment:
```suggestion
                               :placeholder="$props.data.securityTxt.default"
```
Not sure what this placeholder is, probably best to use the default for the item `$props.data.securityTxt.default`

Reply:
My bad, i've changed it on my latest commit, you can check it again :)